### DISCUSSION TITLE
Fluxstore Delivery boy - Support Dokan Plugin

### DISCUSSION THREAD

**Discussion by zijemu** [inspireui/support#5266] [Working on Features]
It's possible to have a fluxstore delivery boy for woocommerce with dokan plugin.

**Comment by JosAJ** [comment]
Already find an add-on that supports DOKAN for home delivery.
This add-on is very good.
https://powerfulwp.com/local-delivery-drivers-for-woocommerce-premium/
And also this one:
https://powerfulwp.com/delivery-drivers-for-woocommerce-multi-vendor-marketplace/
I hope they can do great things with it and be one of the first to sell it.

**Comment by Haniel5** [comment]
Hi, currently FluxStore Delivery Boy app https://1.envato.market/4edvXL  supports both Woocommerce (single vendor), Dokan, and WCFM (WC Lovers) multi vendors too - Very Best!
